### PR TITLE
Rustup

### PR DESCRIPTION
**Pull request by flip1995**
changelog: none

### CODE REVIEW

**Comment by matthiaskrgr** [comment]
@bors r+
thanks?

**Comment by matthiaskrgr** [comment]
Problems with rls: https://github.com/rust-lang/rust-clippy/runs/747024687#step:10:14

File: clippy_lints/src/write.rs [hunk 1]
```diff
@@ -369,7 +369,7 @@ impl Write {
         tts: &TokenStream,
         is_write: bool,
     ) -> (Option<StrLit>, Option<Expr>) {
-        use fmt_macros::{
+        use rustc_parse_format::{
             AlignUnknown, ArgumentImplicitlyIs, ArgumentIs, ArgumentNamed, CountImplied, FormatSpec, ParseMode, Parser,
```
Comment (matthiaskrgr) [inline clippy_lints/src/write.rs:373]: I might have to update `cargo dev ra_setup` due to the renaming.

**Comment by flip1995** [comment]
The RLS failure is because fmt_macros wasn't renamed in RLS yet. I will disable the integration test for now.

**Comment by matthiaskrgr** [comment]
Do we still need the rls test?

I remember already caused problems recently in the past, we might want to reevaluate if it's actually worth the trouble.

**Comment by flip1995** [comment]
I can't remember what the initial policy was for the initial selection of integration test code bases. Some of them were included, because multiple ICEs were detected in them. 

That being said, I think that dependence on compiler internals may be an argument against having the codebase as an integration test 🤔

File: clippy_lints/src/write.rs (hunk 1 above)
Comment (matthiaskrgr) [reply clippy_lints/src/write.rs:373]: Looks like this does not cause any problems. :thinking:

**Pull request merged**